src/lib/constants.ts

```diff
@@ -169,3 +169,64 @@ export const LSP6_DEFAULT_PERMISSIONS = {
 
 export const LSP6_ALL_PERMISSIONS =
   '0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff';
+
+export const COMMON_ABIS = {
+  supportsInterface: [
+    {
+      inputs: [
+        {
+          internalType: 'bytes4',
+          name: 'interfaceId',
+          type: 'bytes4',
+        },
+      ],
+      name: 'supportsInterface',
+      outputs: [
+        {
+          internalType: 'bool',
+          name: '',
+          type: 'bool',
+        },
+      ],
+      stateMutability: 'view',
+      type: 'function',
+    },
+  ],
+};
+
+export const IPFS_GATEWAY_BASE_URL = 'https://ipfs.io/ipfs/'; // 'https://2eff.lukso.dev/ipfs/' not working;
+
+export enum LSPType {
+  LSP3UniversalProfile = 'LSP3',
+  LSP7DigitalAsset = 'LSP7',
+  LSP8IdentifiableDigitalAsset = 'LSP8',
+  LSP9Vault = 'LSP9',
+  Unknown = 'Unknown',
+}
+
+// from @lukso/lsp-smart-contracts v0.7.0, erc725.js should stay independent
+export const INTERFACE_IDS = {
+  ERC165: '0x01ffc9a7',
+  ERC1271: '0x1626ba7e',
+  ERC20: '0x36372b07',
+  ERC223: '0x87d43052',
+  ERC721: '0x80ac58cd',
+  ERC721Metadata: '0x5b5e139f',
+  ERC777: '0xe58e113c',
+  ERC1155: '0xd9b67a26',
+  ERC725X: '0x44c028fe',
+  ERC725Y: '0x714df77c',
+  LSP0ERC725Account: '0xeb6be62e',
+  LSP1UniversalReceiver: '0x6bb56a14',
+  LSP1UniversalReceiverDelegate: '0xa245bbda',
+  LSP6KeyManager: '0xc403d48f',
```

Comment:
Handling versioning for these worries me a bit, what about having an object for these containing the versions or smth?